external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -153,11 +201,7 @@ private[kafka010] case class KafkaSource(
 
```

Comment:
Ok, I see what you were saying, thought you were talking about resetting it in getOffset, not the end passed to getBatch.